.gitattributes

```diff
@@ -0,0 +1,54 @@
+*.doc  diff=astextplain
```

Comment:
Were you able to get the files checked out with \r\n with this file in place? If so, do you know what this was needed vs. the simpler one we tried.

Reply:
I just copy pasted it from CLI repo. And for some reason text=auto stop working and the test failed again, after 30 minutes later I decide just use ignore new line assertion :(